vowpalwabbit/gen_cs_example.h

```diff
@@ -144,7 +144,7 @@ void gen_cs_label(cb_to_cs& c, example& ec, COST_SENSITIVE::label& cs_ld, uint32
 }
 
 template <bool is_learn>
-void gen_cs_example_dr(cb_to_cs& c, example& ec, CB::label& ld, COST_SENSITIVE::label& cs_ld, float clip_p=0.f)
+void gen_cs_example_dr(cb_to_cs& c, example& ec, CB::label& ld, COST_SENSITIVE::label& cs_ld, float /*clip_p*/ =0.f)
```

Comment:
Does the default value still need to be here even though the variable is unused?

Reply:
Yes, it allows the function to be called without the last parameter (as is done in existing code). If I remove it the last parameter becomes required.

Reply:
Yes, it allows the function to be called without the last parameter (as is done in existing code). If I remove it the last parameter becomes required.

Reply:
Makes sense.